### DISCUSSION TITLE
How to get a list of fields that the client requests?

### DISCUSSION THREAD

**Discussion by alexander161198** [ChilliCream/hotchocolate#5835] [Q&A]
Hello!

 How can I get a list of the fields the client is requesting (e.g. in query or in update mutation)? I can get all fields of the requesting model (and do some permimission checks, for example). But I'd like to get the context of the user's request, i.e. list of requested fields.

Thank you for help.

**Comment by PascalSenn** [comment]
@alexander161198  You can use `context.GetSelections(context.ObjectType)` for this. `context` is the `IResolverContext`
